src/Momento.Sdk/Momento.Sdk.csproj

```diff
@@ -6,15 +6,15 @@
 		<LangVersion>latest</LangVersion>
 		<Nullable>enable</Nullable>
 		<!-- Include documentation in build -->
-		<!-- <NoWarn>1591</NoWarn> -->
+		<!-- TODO: Address public-facing documentation -->
+		<NoWarn>1591</NoWarn>
```

Comment:
Updated this since we don't have documentation for all the data structures' response types.